### PR TITLE
[RP-3378]: Added Sample input in CSV Parser plugin.

### PR DESCRIPTION
**Pull request by ashish-madaan-guavus**

### CODE REVIEW

File: transform-plugins/docs/CSVParser-transform.md [hunk 1]
```diff
@@ -19,3 +19,54 @@ Input records with a null input field propagate all other fields and set fields
...
+            "schema": "{\"type\":\"record\",\"name\":\"etlSchemaBody\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}"
+          }
+        ]
+    }
+    
+    
+## Sample Output
+After Parsing the Sample input by CSV parser, The output will be saved to a file in the format of json
+
+    {"id":"id","test1":"test1","test2":"test2","servicetac":"servicetac","operstatus":"operstatus","manager":"manager","itseverity":"itseverity"}
```
Comment (deepankan-bn-guavus) [inline transform-plugins/docs/CSVParser-transform.md]: Please provide output of CSV parser in RDD format.

**Review by deepankan-bn-guavus — COMMENTED** [review]
required changes as mentioned.

File: transform-plugins/docs/CSVParser-transform.md [hunk 2]
```diff
@@ -19,3 +19,57 @@ Input records with a null input field propagate all other fields and set fields
...
+          {
+            "name": "File",
+            "schema": "{\"type\":\"record\",\"name\":\"etlSchemaBody\",\"fields\":[{\"name\":\"offset\",\"type\":\"long\"},{\"name\":\"body\",\"type\":\"string\"}]}"
+          }
+        ]
+    }
+    
+    
+## Sample Output
+After Parsing the Sample input by CSV parser, The output will be saved to a file in the format of json
```
Comment (deepankan-bn-guavus) [inline transform-plugins/docs/CSVParser-transform.md]: please change the description here, it is written as json.

**Review by deepankan-bn-guavus — COMMENTED** [review]
please review my comment.

**Review by deepankan-bn-guavus — APPROVED** [review]

**Review by prashant-gour-guavus — APPROVED** [review]

**Pull request merged**